syntax = "proto3";

message User {
    string username = 1;
    string email = 2;
    string hashed_password = 3;
    string imageUrl = 4;
    string date_of_birth = 6;
    string gender = 7;
    string token = 8;
}

message EmailUser {
    string username = 1;
    string email = 2;
    string imageUrl = 3;
    bool is_active = 4;
    bool is_verified = 5;
    string role = 6;
}

message Otp {
    string email = 1;
    string token = 2;
    string otp = 3;
}